CMakeLists.txt

```diff
@@ -1025,7 +1032,7 @@ if (BUILD_STATIC)
   add_library(cryptopp-static STATIC ${cryptopp_LIBRARY_SOURCES})
   cryptopp_target_compile_properties(cryptopp-static)
   if (NOT ${CMAKE_VERSION} VERSION_LESS "2.8.11")
-    target_include_directories(cryptopp-static PUBLIC $<BUILD_INTERFACE:${SRC_DIR}> $<INSTALL_INTERFACE:include>)
+    target_include_directories(cryptopp-static PUBLIC $<BUILD_INTERFACE:${SRC_DIR}/..> $<INSTALL_INTERFACE:include>)
```

Comment:
That was necessary to make it build for iOS. Unfortunately I can't remember the exact scenario, but it's been building fine for us for a few months now.